pkg/network/tracer/connection/fentry/tracer.go

```diff
@@ -65,6 +69,17 @@ func LoadTracer(config *config.Config, mgrOpts manager.Options, perfHandlerTCP *
 			Name:  "systemprobe_ino",
 			Value: inode,
 		})
+		kprobe.AddBoolConst(&o, ringbufferEnabled, "ringbuffers_enabled")
```

Comment:
There should be no dependency going from `fentry` to `kprobe` or vice versa.

Reply:
moved common to new util package